shared/triage_metadata.go

```diff
@@ -265,6 +265,23 @@ func (tm triageMetadata) Triage(metadata MetadataResults) error {
 	}
 
 	triagedMetadataMap := addToFiles(metadata, filesMap, tm.logger)
-	tm.metadataGithub.wptmetadataGitHubInfo = getWptmetadataGitHubInfo(tm.ctx, tm.githubClient)
+	tm.MetadataGithub.wptmetadataGitHubInfo = getWptmetadataGitHubInfo(tm.ctx, tm.githubClient)
 	return tm.createWPTMetadataPR(triagedMetadataMap)
 }
+
+// GetTriageMetadata returns an instance of triageMetadata struct.
+func GetTriageMetadata(ctx context.Context, git MetadataGithub, logger Logger, httpClient *http.Client) TriageMetadataInterface {
+	return triageMetadata{
+		ctx:            ctx,
+		MetadataGithub: git,
+		logger:         logger,
+		httpClient:     httpClient}
+}
+
+// GetMetadataGithub returns an instance of MetadataGithub struct.
+func GetMetadataGithub(githubClient *github.Client, authorName string, authorEmail string) MetadataGithub {
```

Comment:
Given that MetadataGithub is public (capitalized), I assume this is just a helper to shorten the code needed to construct a MetadataGithub? E.g.

`foo := shared.GetMetadataGithub(gc, author, email)`

Instead of:

`foo := shared.MetadataGithub{githubClient: gc, authorName: author, authorEmail: email}`

?

Reply:
I won't be able to write it as shared.MetadataGithub{githubClient: gc, authorName: author, authorEmail: email} because githubClient, author and email are private fields.